Assets/Scripts/SS3D/SceneManagement/SceneSystem.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Coimbra;
+using Coimbra.Services.Events;
+using SS3D.Application;
+using SS3D.Application.Events;
+using SS3D.Data.AssetDatabases;
+using SS3D.Data.Generated;
+using SS3D.Logging;
+using System;
+
+// ReSharper disable ConditionIsAlwaysTrueOrFalse
+namespace SS3D.SceneManagement
+{
+	public sealed class SceneSystem : Core.Behaviours.System
```

Comment:
Please explain a bit more in doc the current and future purpose of this system.

Reply:
Done